web/app/js/components/ResourceList.jsx

```diff
@@ -6,113 +6,46 @@ import MetricsTable from './MetricsTable.jsx';
 import PageHeader from './PageHeader.jsx';
 import { processRollupMetrics } from './util/MetricUtils.js';
 import React from 'react';
+import withREST from './util/RESTBridge.jsx';
 import './../../css/list.css';
 import 'whatwg-fetch';
 
-export default class ResourceList extends React.Component {
-  constructor(props) {
-    super(props);
-    this.api = this.props.api;
-    this.handleApiError = this.handleApiError.bind(this);
-    this.loadFromServer = this.loadFromServer.bind(this);
-
-    this.state = this.getInitialState(this.props);
-  }
-
-  getInitialState() {
-    return {
-      pollingInterval: 2000, // TODO: poll based on metricsWindow size
-      metrics: [],
-      pendingRequests: false,
-      loaded: false,
-      error: ''
-    };
-  }
-
-  componentDidMount() {
-    this.startServerPolling(this.props.resource);
-  }
-
-  componentWillReceiveProps(newProps) {
```

Comment:
when switching between resource pages on this branch, the old state seems to carry over to the new until the next data load - we should figure out how to clear the table / prevent the old data from carrying over